Packs/QRadar/Integrations/QRadar_v3/QRadar_v3.py

```diff
@@ -1127,31 +1130,36 @@ def get_remote_events(client: Client,
             changed_ids_ctx.append(offense_id)
             status = QueryStatus.SUCCESS.value
         except Exception as e:
-            # getting results failed, move back to queried queue to be queried again
+            # move back to queried queue
             del offenses_finished[offense_id]
             changed_ids_ctx.append(offense_id)
             offenses_queried[offense_id] = QueryStatus.ERROR.value
             status = QueryStatus.ERROR.value
             print_debug_msg(f'No results for {offense_id}. Error: {e}. Stopping execution')
             time.sleep(FAILURE_SLEEP)
-    elif offense_id in offenses_queried:  # if our offense is in the queried list, we will get the result
+
+    elif offense_id in offenses_queried:
         search_id = offenses_queried[offense_id]
-        events, status = poll_offense_events(client, search_id, should_get_events=True, offense_id=int(offense_id))
+        events, status = poll_offense_events(client, search_id, True, int(offense_id))
```

Comment:
I was actually thinking it is better to name all the arguments we pass, in a complex function like this. makes it more readble.